SPECS/syslog-ng/syslog-ng.spec

```diff
@@ -0,0 +1,104 @@
+Summary:	Next generation system logger facilty
+Name:		syslog-ng
+Version:	3.6.2
+Release:	1%{dist}
+License:	GPL + LGPL
+URL:		https://www.balabit.com/network-security/syslog-ng/opensource-logging-system
+Group:		System Environment/Daemons
+Vendor:		VMware, Inc.
+Distribution: 	Photon
+Source0:	http://my.balabit.com/downloads/syslog-ng/open-source-edition/3.6.2/source/syslog-ng_3.6.2.tar.gz
```

Comment:
Can you replace syslog-ng_3.6.2.tar.gz with name and version tags as in other spec files.
Also replace folder name 3.6.2/ in the path with version tag.